topology/Louisiana State University/LSU - Deep Bayou/LSU-Deep_Bayou.yaml

```diff
@@ -0,0 +1,32 @@
+Production: true
+SupportCenter: Self Supported
+GroupID: 1105
+GroupDescription: This is the LSU - Deep Bayou cluster
+Resources:
+  LSU-Deep_Bayou:
```

Comment:
```suggestion
  LSU-DB-CE1:
```

Maybe something like this to more closely match the first bit of the FQDN?

Reply:
Hmm I think I like this suggestion. @jeffreynpeterson what are your thoughts? We can update the naming convention document if you agree.

Reply:
matching the FQDN is good, we will still just have to make the call as to when to shorten or not the name for that depending on site.